### PR TITLE
Use the SKU sellerId in BuyButton instead of set the seller in a hard code way.

### PR DESCRIPTION
**Pull request by ArthurSampaio**
#### What is the purpose of this pull request?

Change the default `seller` props in `BuyButton` to a value taken from product/sku.

#### What problem is this solving?

Pass as BuyButton's props the sellerId taken from query.

#### How should this be manually tested?
https://arthur--storecomponents.myvtex.com

#### Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [1.5.0] - 2018-10-25
```
Comment (lucasecdb) [inline CHANGELOG.md]: This was already released?

Comment (ArthurSampaio) [reply CHANGELOG.md]: No sorry, I had not paid attention to the base format of the changelog.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
```
Comment (lucasecdb) [inline CHANGELOG.md]: Remove the empty line above :smile:

Comment (ArthurSampaio) [reply CHANGELOG.md]: :disappointed: done!

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
+- Added sellerId in `BuyButton` component.
```
Comment (brunojdo) [inline CHANGELOG.md]: `Add` always write in the infinitive

**Pull request merged**